Tensile/BenchmarkProblems.py

```diff
@@ -127,7 +127,7 @@ def writeBenchmarkFiles(stepBaseDir, solutions, problemSizes, \
 
     # get unique kernels and kernel helpers
     for solution in Utils.tqdm(solutions, desc="Finding unique solutions"):
-        solutionKernels = solution.getKernels()
+        solutionKernels = [solution.getKernels()]
```

Comment:
Won't this necessarily be a single element list? In which case we can remove the for loop?

Reply:
Fixed